app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -104,7 +104,9 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
     private fun registerSessionObserver() {
         components.sessionManager.register(object : SessionManager.Observer {
             override fun onSessionSelected(session: Session) {
-                showBrowserScreenForCurrentSession()
+                if (!session.isCustomTabSession()) {
+                    showBrowserScreenForCurrentSession()
+                }
```

Comment:
This is a bug in `SessionManager`. A custom tab should never get selected in the first place.